.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,226 @@
+                echo "Deploy apachebench pod and run test"
+                cd .pipelines/cni/scale-test-templates
+                kubectl apply -f apache.yaml
+                echo "wait for pod to become ready"
+                sleep 5s
+                kubectl get pod -owide
+                mkdir results
+                cd results
+                AB_POD=$(kubectl get pod -l app=apachebench | grep apachebench | awk '{print $1}')
+                kubectl exec -it $AB_POD -- ab -n 100000 -c 3 -r http://real-svc-00001.scale-test/ >> "ab_100krequests_60kpods.log"
+                echo "running k top node" 
+                kubectl top node >> "node_cpu_and_mem.log"
+                echo "running k top pod"
+                kubectl top pod -A | grep cilium >> "pod_cpu_and_mem.log"
+                echo "Logs will be available as a build artifact"
+                ARTIFACT_DIR=$(Build.ArtifactStagingDirectory)/results/
+                echo $ARTIFACT_DIR
+                sudo rm -rf $ARTIFACT_DIR
+                sudo mkdir $ARTIFACT_DIR
+                cd ..
+                sudo cp ./results/* $ARTIFACT_DIR
+            name: "TestLBServices"
+            displayName: "Apachebench testing"
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: results
+              pathtoPublish: "$(Build.ArtifactStagingDirectory)/results"
+            condition: always()
+            name: "PublishResults"
+            displayName: "Apachebench Result Artifacts"
+  - stage: scale_down_cluster
+    displayName: "Scale Down Cluster"
+    jobs:
+      - job: scale_down
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                echo "Scaling to 5 nodes"
+                vmss_name=$(az vmss list -g MC_${CLUSTER}_${CLUSTER}_$(LOCATION) --query "[].name" -o tsv)
+                make -C ./hack/aks scale-vmss AZCLI=az CLUSTER=${CLUSTER} REGION=$(LOCATION) VMSS_NAME=$vmss_name NODE_COUNT=5
+                kubectl get node
+            name: "ScaleDown"
+            displayName: "Scale down to 5 Nodes"
+  - stage: delete_test_namespaces
+    displayName: "Delete Test Namespaces"
+    jobs:
+      - job: delete_namespaces
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+        - task: AzureCLI@1
+          inputs:
+            azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+            scriptLocation: "inlineScript"
+            scriptType: "bash"
+            addSpnToEnvironment: true
+            inlineScript: |
+              echo "scale deployment and test network policies"
```

Comment:
Seems incorrect based on what stage is doing.

Reply:
thanks, good catch